api/access/server.go

```diff
@@ -130,8 +129,25 @@ func (s *Server) GetBlockByHeight(ctx context.Context, in *access.GetBlockByHeig
 	return nil, errors.New("not implemented")
 }
 
-func (s *Server) GetCollectionByID(ctx context.Context, in *access.GetCollectionByIDRequest) (*access.CollectionResponse, error) {
-	return nil, errors.New("not implemented")
+func (s *Server) GetCollectionByID(_ context.Context, in *access.GetCollectionByIDRequest) (*access.CollectionResponse, error) {
+	id := flow.HashToID(in.Id)
```

Comment:
I would prefer to use `collID` here. A blank `id` should rarely be used, imo, as you could then easily make the mistake of passing a transaction ID to the collection function (they have the same type), etc.